konfera/templates/konfera/event/cfp_form.html

```diff
@@ -2,17 +2,31 @@
 {% load i18n %}
 
 {% block event_details %}
+  {% block top_cfp %}{% endblock %}
+    {% if event.cfp_open %}
+      <form method="POST" action="" class="cfp">{% csrf_token %}
+        <h2>{% trans "Who are you? Please introduce yourself." %}</h2>
+          {{ speaker_form.as_p }}
+        <h2>{% trans "What would you like to talk about?" %}</h2>
+          {{ talk_form.as_p }}
+        <button type="submit" class="btn btn-default btn-lg center-block">
+          <span class="glyphicon glyphicon-thumbs-up" aria-hidden="true"></span> {% trans "Submit proposal" %}
+        </button>
+      </form>
 
-    {% block top_cfp %}{% endblock %}
-    <form method="POST" action="" class="cfp">{% csrf_token %}
-      <h2>{% trans "Who are you? Please introduce yourself." %}</h2>
-        {{ speaker_form.as_p }}
-      <h2>{% trans "What would you like to talk about?" %}</h2>
-        {{ talk_form.as_p }}
-      <button type="submit" class="btn btn-default btn-lg center-block">
-        <span class="glyphicon glyphicon-thumbs-up" aria-hidden="true"></span> {% trans "Submit proposal" %}
-      </button>
-    </form>
-    {% block bottom_cfp %}{% endblock %}
-
+    {% else %}
+      <div class="row">
+        <div class="col-md-8">
+          <p>
+            {% trans "Thank you for your interest, but unfortunately call for proposals deadline was at: "
+            {{ event.cfp_end }}.
+            <br/>
+            {% trans "However, if you feel like you have to submit your talk anyway, feel free to contact us:"
+            {{ event.contact_email }}
+            <br/>
```

Comment:
Using `<br/>` for *design* purposes is an antipattern. If it looks strange without it, please update the *css* files.